app/components/em-list.js

```diff
@@ -0,0 +1,75 @@
+import Ember from 'ember';
+import { task, timeout } from 'ember-concurrency';
+import filterByQuery from 'ember-cli-filter-by-query/util/filter';
+
+const {
+  Component,
+  computed
+} = Ember;
+
+const QUERY_DEBOUNCE = 200;
+
+export default Component.extend({
+  tagName: 'table',
+  classNames: 'table table-packages',
+
+  init() {
+    this._super(...arguments);
+    this._filterList(this.get('query'));
+    Ember.run.scheduleOnce('afterRender', () => {
```

Comment:
@stefanpenner thank you for your review. I agree with your comments and have removed the related code in https://github.com/gcollazo/ember-cli-addon-search/pull/92